src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -36,10 +44,11 @@ const PageLayoutWrapper = ({
 
   const scriptSwitchId = pathOr('', ['scriptSwitchId'], pageData);
   const renderScriptSwitch = pathOr(true, ['renderScriptSwitch'], pageData);
-  const isErrorPage = [404, 500].includes(status);
-  const pageType = isErrorPage
-    ? 'WS-ERROR-PAGE'
-    : path<string>(['metadata', 'type'], pageData);
+
+  const isErrorPage = ![200].includes(status) || !status;
+  const cpsPageType = pageData?.metadata?.type;
+  const otherPageType = pageData?.pageType;
+  const pageType = isErrorPage ? 'WS-ERROR-PAGE' : cpsPageType || otherPageType;
```

Comment:
```suggestion
  const pageType = isErrorPage ? 'WS-ERROR-PAGE' : metadataType || fallbackPageType;
```

Reply:
Question about whether or not we need to logic for error pages, if we're no longer going to track them in Web Vitals.

Reply:
removed logic to send WS-ERROR-PAGE as page type